webui/react/src/components/OverviewStats.tsx

```diff
@@ -3,14 +3,15 @@ import React, { PropsWithChildren } from 'react';
 import css from './OverviewStats.module.scss';
 
 interface Props {
+  clickable?: boolean
```

Comment:
just noting that this change would not be needed if we consumed the hook directly